website/docs/using-podman-desktop-behind-a-proxy/index.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 3
+title: Using behind a proxy
+description: Using Podman Desktop behind a proxy requiring custom Certificate Authorities (CA).
+tags: [podman-desktop, installing, windows, macos, linux]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos, linux]
+---
+
+# Using Podman Desktop behind a proxy requiring custom Certificate Authorities (CA)
+
+You can use Podman Desktop behind a proxy requiring custom Certificate Authorities (CA), such as self-signed certificates.
+
+#### Prerequisites
+
+* `<your-custom-ca.pem>`: Your proxy Certificate Authorities (CA), in Privacy-Enhanced Mail (PEM) format.
+* `<your.proxy.tld>`: Your proxy URL.
+
+#### Procedure
+
+1. (On Windows and macOS) Open a shell prompt on the Podman machine:
```

Comment:
This is a false positive:

> [RedHat.CaseSensitiveTerms] Use 'Windows Server' rather than 'Windows'.

* Issue reported: https://github.com/redhat-documentation/vale-at-red-hat/issues/458